### ISSUE TITLE
[BUG]: Player Permission error?

### ISSUE THREAD

**Issue by NerdyViking** [ZotyDev/objects-interactions-fx#25]
### Discord Username

RealNerdyViking

### Browsers

Chrome, Native Foundry Client

### What Happened?

Players are granted all permissions and set to Trusted Player. As a GM when I place a new token down on the battle map, with no other modules except OIF and dependencies enabled, I get this error.

![image](https://github.com/ZotyDev/objects-interactions-fx/assets/63491456/90f07565-183f-4de5-9460-4dd6959923ce)

### Reproduction Steps

```bash
1. Have player logged in.
2. Place new token down on battle map with them activated on that map.
```

**Comment by ZotyDev** [comment]
Hi! Thanks for reaching out about this bug!
I'm searching for a fix and will release it asap.

**Comment by NerdyViking** [comment]
As an update to this. I created an entirely new world with just the required AND recommended modules on and it seems to be working fine. To reconfirm, I disabled all modules on my primary world and only loaded the exact same setup and I continue to receive the error.

This leads me to suspect that there might be some incompatibilities with certain modules I have installed that perhaps alter a setting when activated. Would you like a list of all my modules, or just not gonna worry about it?

**Comment by ZotyDev** [comment]
I would like a list of the modules please, this issue will probably be fixed on v2.0.0 since almost the entire code changed but it can be helpful as reference for future bugs.

**Comment by NerdyViking** [comment]
![image](https://github.com/ZotyDev/objects-interactions-fx/assets/63491456/80436449-3d63-48f2-988e-1e8da3fe1773)
![image](https://github.com/ZotyDev/objects-interactions-fx/assets/63491456/487f22ef-4fdc-41e3-87ca-087fac4ed85b)
![image](https://github.com/ZotyDev/objects-interactions-fx/assets/63491456/0968c952-86ff-4211-8131-4f0d923472cb)
![image](https://github.com/ZotyDev/objects-interactions-fx/assets/63491456/b3db21ae-7268-40e2-a064-51f8a77ef9ad)

**Comment by NerdyViking** [comment]
@ZotyDev I did some more digging for you, mostly cuz I'm curious. Anyway, It seems the error only appears after I try to set the settings to add ammunition or throwables to target inventory. Seems to work fine with item piles on a miss, but won't work to add to target.

That all being said, I tested it shooting a crossbow both as the gm and as the player and the items are successfully added to the inventory as a gm, but not as a player. Though, once the error begins appearing, not alteration of settings will fix it and make it not appear again.

Hope this extra info is helpful.
